app/src/main/java/com/codesoom/assignment/controllers/SessionController.java

```diff
@@ -0,0 +1,37 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.application.AuthenticationService;
+import com.codesoom.assignment.dto.SessionResponseData;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * session에 대한 요청을 처리
+ */
+@RestController
+@RequestMapping("/session")
+public class SessionController {
+    private AuthenticationService authenticationService;
+
+    public SessionController(AuthenticationService authenticationService) {
+        this.authenticationService = authenticationService;
+    }
+
+    /**
+     * 사용자 로그인시 정보를 보호
+     *
+     * @return 사용자의 암호화된 정보
+     */
```

Comment:
이 JavaDoc의 문장이 어떤 것을 의미하는지 잘 모르겠어요. 잘 모르겠을 때에는 메소드의 JavaDoc은 메소드의 "리턴값"을 설명해주는 방식으로 작성해 주세요.

```suggestion
    /**
     * 사용자를 로그인 처리하고, 세션 응답 데이터를 리턴합니다.
     *
     * @return 세션 응답 데이터
     */
```

Reply:
javadoc 작성이 어려웠는데 메소드의 리턴값을 설명해주는 방식으로 작성하면 되는군요. 좀 더 동료들에게 잘 전달될 수 있게 쓰는 방법을 알아봐야겠네요.

Reply:
JavaDoc은 리턴값을 설명해주는 것이 기본입니다. 아래의 스크린샷은 `String.java` 파일의 일부를 찍은 거에요.

![image](https://user-images.githubusercontent.com/1855714/148056722-ffe530af-8853-46ad-b975-cb4e04bc40b0.png)